### PR TITLE
Create Psammead Media Player

### LINKED ISSUE #1933: Move Video component from simorgh to psammead
**Is your feature request related to a problem? Please describe.**
The Video component should not be in this repo, it should be in bbc/psammead

https://github.com/bbc/psammead/blob/latest/CONTRIBUTING.md

**Describe the solution you'd like**
https://www.youtube.com/watch?v=hdcTmpvDO0I

**Describe alternatives you've considered**
None

**Testing notes**
[Tester to complete]

Dev insight: It's not currently used, so minimal.

**Additional context**
n/a

### PR DESCRIPTION
**Pull request by 12**
Resolves bbc/simorgh#1933

**Overall change:** Moves Media Player component from http://github.com/bbc/simorgh/tree/latest/src/app/containers/MediaPlayer to a ***new*** Psammead component, named `psammead-media-player`

**Code changes:**

- Exports both a Canonical and AMP media player.
- Adds snapshots for each exported component and different variants of each
- Initialises the new module

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

File: packages/components/psammead-media-player/README.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
+# (psammead-media-player) &middot;
```
Comment (dr3) [inline packages/components/psammead-media-player/README.md]:
```suggestion
# ⛔️ This is an alpha component ⛔️

This component is currently tagged as alpha and is not suitable for production use. Following the passing of an accessibility review this component will be marked as ready for production and the alpha tag removed.

# (psammead-media-player) &middot; [![Known Vulnerabilities](https://snyk.io/test/github/bbc/psammead/badge.svg?targetFile=packages%2Fcomponents%2Fpsammead-brand%2Fpackage.json)](https://snyk.io/test/github/bbc/psammead?targetFile=packages%2Fcomponents%2Fpsammead-brand%2Fpackage.json) [![Dependency Status](https://david-dm.org/bbc/psammead.svg?path=packages/components/psammead-media-player)](https://david-dm.org/bbc/psammead?path=packages/components/psammead-media-player) [![peerDependencies Status](https://david-dm.org/bbc/psammead/peer-status.svg?path=packages/components/psammead-media-player)](https://david-dm.org/bbc/psammead?path=packages/components/psammead-media-player&type=peer) [![Storybook](https://raw.githubusercontent.com/storybooks/brand/master/badge/badge-storybook.svg?sanitize=true)](https://bbc.github.io/psammead/?path=/story/brand--default) [![GitHub license](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://github.com/bbc/psammead/blob/latest/LICENSE) [![npm version](https://img.shields.io/npm/v/@bbc/psammead-media-player.svg)](https://www.npmjs.com/package/@bbc/psammead-media-player) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://github.com/bbc/psammead/blob/latest/CONTRIBUTING.md) 
```

File: packages/components/psammead-media-player/package.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@bbc/psammead-media-player",
+  "version": "0.1.0-alpha.0",
```
Comment (dr3) [inline packages/components/psammead-media-player/package.json]: Should be `1.0.0-alpha.0`

File: packages/components/psammead-media-player/src/Placeholder/index.jsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+`;
+
+const Placeholder = ({ onClick, src }) => (
+  <StyledPlaceholder onClick={onClick}>
+    <Image alt="Image Alt" src={src} />
```
Comment (dr3) [inline packages/components/psammead-media-player/src/Placeholder/index.jsx:17]: Any issue to properly set the alt?

File: packages/components/psammead-media-player/src/__snapshots__/index.test.jsx.snap [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+exports[`Media Player: AMP Entry renders a landscape container with an amp-iframe and nested amp-img 1`] = `
+.c0 {
+  padding-top: 56.25%;
+  position: relative;
+  overflow: hidden;
+}
+
+<div
+  class="c0"
+  orientation="Landscape"
```
Comment (dr3) [inline packages/components/psammead-media-player/src/__snapshots__/index.test.jsx.snap]: lol its passing `orientation="Landscape"` to the div? Is `orientation` a html thing? Might need to rename it as to not pass it to the dom

File: packages/components/psammead-media-player/src/index.jsx [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+};
+
+CanonicalMediaPlayer.defaultProps = {
+  showPlaceholder: true,
+  placeholderSrc: null,
+  orientation: 'Landscape',
+};
+
+AmpMediaPlayer.propTypes = {
+  orientation: oneOf(['Portrait', 'Landscape']),
```
Comment (dr3) [inline packages/components/psammead-media-player/src/index.jsx]: Does this need to have capitals? Bit gross imo

**Review by dr3 — CHANGES_REQUESTED** [review]
really nice, just a couple things

File: packages/components/psammead-media-player/src/index.jsx (hunk 5 above)
Comment (12) [reply packages/components/psammead-media-player/src/index.jsx]: Keeping consistent with the fact Ares gives us ‘Portrait’ for portrait

File: packages/components/psammead-media-player/src/Placeholder/index.jsx (hunk 3 above)
Comment (12) [reply packages/components/psammead-media-player/src/Placeholder/index.jsx:17]: #1840

File: packages/components/psammead-media-player/src/__snapshots__/index.test.jsx.snap (hunk 4 above)
Comment (12) [reply packages/components/psammead-media-player/src/__snapshots__/index.test.jsx.snap]: haha oh yeah, fixed

File: packages/components/psammead-media-player/README.md [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
...io/test/github/bbc/psammead/badge.svg?targetFile=packages%2Fcomponents%2Fpsammead-brand%2Fpackage.json)](https://snyk.io/test/github/bbc/psammead?targetFile=packages%2Fcomponents%2Fpsammead-brand%2Fpackage.json) [![Dependency Status](https://david-dm.org/bbc/psammead.svg?path=packages/components/psammead-media-player)](https://david-dm.org/bbc/psammead?path=packages/components/psammead-media-player) [![peerDependencies Status](https://david-dm.org/bbc/psammead/peer-status.svg?path=packages/components/psammead-media-player)](https://david-dm.org/bbc/psammead?path=packages/components/psammead-media-player&type=peer) [![Storybook](https://raw.githubusercontent.com/storybooks/brand/master/badge/badge-storybook.svg?sanitize=true)](https://bbc.github.io/psammead/?path=/story/brand--default) [![GitHub license](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://github.com/bbc/psammead/blob/latest/LICENSE) [![npm version](https://img.shields.io/npm/v/@bbc/psammead-media-player.svg)](https://www.npmjs.com/package/@bbc/psammead-media-player) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://github.com/bbc/psammead/blob/latest/CONTRIBUTING.md)
```
Comment (dr3) [inline packages/components/psammead-media-player/README.md]:
```suggestion
# psammead-media-player &middot; [![Known Vulnerabilities](https://snyk.io/test/github/bbc/psammead/badge.svg?targetFile=packages%2Fcomponents%2Fpsammead-brand%2Fpackage.json)](https://snyk.io/test/github/bbc/psammead?targetFile=packages%2Fcomponents%2Fpsammead-brand%2Fpackage.json) [![Dependency Status](https://david-dm.org/bbc/psammead.svg?path=packages/components/psammead-media-player)](https://david-dm.org/bbc/psammead?path=packages/components/psammead-media-player) [![peerDependencies Status](https://david-dm.org/bbc/psammead/peer-status.svg?path=packages/components/psammead-media-player)](https://david-dm.org/bbc/psammead?path=packages/components/psammead-media-player&type=peer) [![Storybook](https://raw.githubusercontent.com/storybooks/brand/master/badge/badge-storybook.svg?sanitize=true)](https://bbc.github.io/psammead/?path=/story/brand--default) [![GitHub license](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://github.com/bbc/psammead/blob/latest/LICENSE) [![npm version](https://img.shields.io/npm/v/@bbc/psammead-media-player.svg)](https://www.npmjs.com/package/@bbc/psammead-media-player) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://github.com/bbc/psammead/blob/latest/CONTRIBUTING.md) 
```

File: packages/components/psammead-media-player/src/Amp/index.jsx [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+    <Fragment>
+      <AmpHead />
+      <amp-iframe
+        sandbox="allow-scripts allow-same-origin"
+        layout="fill"
+        frameborder="0"
+        src={src}
+        allowfullscreen
+      >
+        <amp-img layout="fill" src={placeholderSrc} placeholder />
```
Comment (dr3) [inline packages/components/psammead-media-player/src/Amp/index.jsx:26]: `placeholderSrc` seems pretty vital here. Should it be `{placeholderSrc && <amp-img />}` so it doesnt render it if theres no image?

**Review by dr3 — CHANGES_REQUESTED** [review]
Great work, really nice package, just 1 small thing

File: packages/components/psammead-media-player/src/Amp/index.jsx (hunk 7 above)
Comment (12) [reply packages/components/psammead-media-player/src/Amp/index.jsx:26]: It should be a required prop so this shouldn't be necessary. Have updated proptypes.

**Review by dr3 — APPROVED** [review]

File: packages/components/psammead-media-player/src/index.jsx (hunk 5 above)
Comment (12) [reply packages/components/psammead-media-player/src/index.jsx]: Have changed this prop to be named `portrait` and accepts a boolean.

**Review by ryanmccombe — APPROVED** [review]

**Pull request merged**